cmd/root.go

```diff
@@ -67,13 +67,15 @@ func Execute() {
 func init() {
 	rootCmd.PersistentFlags().BoolP("debug", "d", false, "Enable or disable debug mode")
 	rootCmd.PersistentFlags().BoolP("parallel", "p", true, "Fetch clones in parallel")
+	rootCmd.PersistentFlags().Bool("commands", true, "Run post-commands")
 	rootCmd.PersistentFlags().
 		StringP("gilt-dir", "c", "~/.gilt/clone", "Path to Gilt's clone dir")
 	rootCmd.PersistentFlags().
 		StringP("gilt-file", "f", "Giltfile.yaml", "Path to config file")
 
 	_ = viper.BindPFlag("debug", rootCmd.PersistentFlags().Lookup("debug"))
 	_ = viper.BindPFlag("parallel", rootCmd.PersistentFlags().Lookup("parallel"))
+	_ = viper.BindPFlag("commands", rootCmd.PersistentFlags().Lookup("commands"))
```

Comment:
I'm a little wobbly with Viper, so while I would love for it to be `--commands` and `--no-commands`, I guess this is how it wants to be done?

¯\\\_(ツ)_/¯

Reply:
Yeah, I think it starts getting weird when we use a bool flag with a default truthy value.  Typically, the bool flag is `false` so the presence of the flag makes sense.

What if we instead did a bool flag of `--no-commands` ?   Otherwise, it seems kidna janky to need to pass true or false to a bool subcmd :/

Reply:
we could also get fancy with stuff like `--with=commands,x,y,z` `--without=commands` to mix and match to our wildest future desires.